src/codemodder/semgrep.py

```diff
@@ -71,7 +71,29 @@ def from_sarif(
         for location in sarif_result["locations"]:
             artifact_location = SemgrepLocation.from_sarif(location)
             locations.append(artifact_location)
-        return cls(rule_id=rule_id, locations=locations)
+        all_flows: list[list[Location]] = [
+            [
+                SemgrepLocation.from_sarif(locations.get("location"))
+                for locations in threadflow.get("locations", {})
+            ]
+            for codeflow in sarif_result.get("codeFlows", {})
+            for threadflow in codeflow.get("threadFlows", {})
+        ]
+        related_locations: list[LocationWithMessage] = []
+        if "relatedLocations" in sarif_result.keys():
```

Comment:
Small thing but this is redundant:
```suggestion
        if "relatedLocations" in sarif_result:
```